operator/controllers/keptnworkloadinstance/controller.go

```diff
@@ -297,28 +297,41 @@ func (r *KeptnWorkloadInstanceReconciler) getAppVersion(ctx context.Context, app
 
 func (r *KeptnWorkloadInstanceReconciler) getAppVersionForWorkloadInstance(ctx context.Context, wli *klcv1alpha1.KeptnWorkloadInstance) (bool, klcv1alpha1.KeptnAppVersion, error) {
 	apps := &klcv1alpha1.KeptnAppVersionList{}
+
 	if err := r.Client.List(ctx, apps, client.InNamespace(wli.Namespace)); err != nil {
 		return false, klcv1alpha1.KeptnAppVersion{}, err
 	}
 	latestVersion := klcv1alpha1.KeptnAppVersion{}
 	for _, app := range apps.Items {
+		r.Log.Info("DEBUG: " + app.Spec.AppName + "==" + wli.Spec.AppName)
```

Comment:
should we remove this log entry? this might lead to very verbose logs, so i suggest to produce log output only for matching app versions